### DISCUSSION TITLE
v1.1特性

### DISCUSSION THREAD

**Discussion by pengwon** [boringhex-top/ch347-hs-usb-bridge#5] [Ideas]
1. 支持多种接口电平
2. 接口分离
3. 通过程序切换功能模式
4. typec

**Comment by pengwon** [comment]
接口电平：
1. 3.3V
2. 2.5V
3. 1.8V
4. 外部输入

是否已经可以满足大部分需求？

**Comment by My-Code-GitHub** [comment]
1.8V和2.5V只有调试系统板载电路时候用的比较多可以保留，如果单独连接外设还是3.3V和5V多一些

**Reply by pengwon** [reply]
是的，之前主要是调试fpga的朋友提出，还有一些低功耗单片机，系统可能用2.5V。

**Comment by My-Code-GitHub** [comment]
> 1. 支持多种接口电平
> 2. 接口分离
> 3. 通过程序切换功能模式
> 4. typec

要不要考虑一个虚拟示波器、虚拟逻辑分析仪的功能，这个想法显示嘛？

**Reply by pengwon** [reply]
这个功能需要变化太大，后面新开一个项目，用rpi搞
